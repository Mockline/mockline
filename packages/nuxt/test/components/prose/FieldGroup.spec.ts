import { describe, it, expect } from 'vitest'
// @ts-ignore
import { MFieldGroup } from '#components'
import type { TypeOf } from "zod";
import ComponentRender from "../../component-render";

describe('CodeIcon', () => {
  /*const props = [
    {
      name: 'analytics',
      type: 'string',
      required: true,
    },
    {
      name: 'blob',
      type: 'string',
      required: false,
    }
  ]*/
  it.each([
    ['with name', { props: { name: 'analytics' } }],
  ])('renders %s correctly', async (nameOrHtml, options: TypeOf<typeof MFieldGroup.props>) => {
    const html = await ComponentRender(nameOrHtml, options, MFieldGroup)
    expect(html).toMatchSnapshot()
  })
})
