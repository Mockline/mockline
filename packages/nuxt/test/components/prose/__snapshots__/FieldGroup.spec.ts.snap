// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CodeIcon > renders with name correctly 1`] = `"<div class="my-5 divide-y divide-(--color-border-primary) *:not-last:pb-5" name="analytics"></div>"`;
